utils/build/util.go

```diff
@@ -187,7 +187,7 @@ func ExpandPackagesNoVendor(patterns []string) []string {
 	}
 	if expand {
 		cmd := GoTool("list", patterns...)
-		out, err := cmd.CombinedOutput()
+		out, err := cmd.Output()
```

Comment:
`CombinedOutput` returns both stdout and stderr, but `Output` only returns stdout.
By the way, `ExpandPackagesNoVendor` will be removed in the next PR because Klayn will not contain `vendor` directory anymore.